### PR TITLE
Fixes for using PostgreSQL

### PR DESCRIPTION
**Pull request by matrss**
Created enum types should also be dropped on downgrade. Dropping tables must be committed to become visible.

**Purpose of PR?**:

Fixes #

**Does this PR introduce a breaking change?**

**If the changes in this PR are manually verified, list down the scenarios covered:**:

**Additional information for reviewer?** :
_Mention if this PR is part of any design or a continuation of previous PRs_

**Does this PR results in some Documentation changes?**
_If yes, include the list of Documentation changes_

**Checklist:**
- [ ] Bug fix. Fixes #<issue number>
- [ ] New feature (Non-API breaking changes that adds functionality)
- [ ] PR Title follows the convention of  `<type>: <subject>`
- [ ] Commit has unit tests

### CODE REVIEW

**Review by ReimarBauer — APPROVED** [review]
I expect that his works. I can verify this again later when we have merged both to develop.

File: mslib/mscolab/migrations/versions/92eaba86a92e_to_version_8_3_5_initial_migration.py [hunk 1]
```diff
@@ -81,7 +81,9 @@ def upgrade():
 def downgrade():
     # ### commands auto generated by Alembic - please adjust! ###
     op.drop_table('permissions')
+    sa.Enum(name='access_level').drop(op.get_bind(), checkfirst=False)
     op.drop_table('messages')
+    sa.Enum(name='messagetype').drop(op.get_bind(), checkfirst=False)
```
Comment (ReimarBauer) [inline mslib/mscolab/migrations/versions/92eaba86a92e_to_version_8_3_5_initial_migration.py:86]: I just looked into the model definition: "message_type"

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
I think it is  "message_type" please double check

File: mslib/mscolab/migrations/versions/92eaba86a92e_to_version_8_3_5_initial_migration.py (hunk 1 above)
Comment (matrss) [reply mslib/mscolab/migrations/versions/92eaba86a92e_to_version_8_3_5_initial_migration.py:86]: The column that uses the type is called `message_type`, but the enum type in the database is created with the name `messagetype`. You can see that here: https://github.com/Open-MSS/MSS/blob/c60241503e9e576d95b01c713dc55e11326d450b/mslib/mscolab/migrations/versions/92eaba86a92e_to_version_8_3_5_initial_migration.py#L61, the relevant part being `name='messagetype'`.

Comment (matrss) [reply mslib/mscolab/migrations/versions/92eaba86a92e_to_version_8_3_5_initial_migration.py:86]: I think it derives that name from simply lowercasing the class name of the enum defined here: https://github.com/Open-MSS/MSS/blob/c60241503e9e576d95b01c713dc55e11326d450b/mslib/mscolab/message_type.py#L28

**Review by ReimarBauer — APPROVED** [review]
ok, thx.

**Pull request merged**